packages/splits-kit/src/components/CreateSplit/CreateSplitForm.tsx

```diff
@@ -0,0 +1,204 @@
+import { useCallback } from 'react'
+import { useCreateSplit } from '@0xsplits/splits-sdk-react'
+import { CreateSplitConfig } from '@0xsplits/splits-sdk'
+import { useForm, FormProvider } from 'react-hook-form'
+import { useAccount, useNetwork } from 'wagmi'
+import { sum, uniq } from 'lodash'
+
+import { ControllerSelector } from '../CreateSplit/ControllerSelector'
+import { CHAIN_INFO, SupportedChainId } from '../../constants/chains'
+import { IAddress, Recipient, CreateSplitForm } from '../../types'
+import RecipientSetter from '../CreateSplit/RecipientSetter'
+import NumberSelectInput from '../inputs/NumberSelectInput'
+import { getNativeTokenSymbol } from '../../utils/display'
+import { getSplitRouterParams } from '../../utils/splits'
+import InputRow from '../inputs/InputRow'
+import Tooltip from '../util/Tooltip'
+import Button from '../util/Button'
+import Link from '../util/Link'
+import { Log } from 'viem'
+
+const CreateCreateSplitForm = ({
+  chainId,
+  defaultDistributorFee,
+  defaultRecipients,
+  defaultController,
+  defaultDistributorFeeOptions,
+  onSuccess,
+}: {
+  chainId: SupportedChainId
+  defaultDistributorFee: number
+  defaultController: IAddress
+  defaultRecipients: Recipient[]
+  defaultDistributorFeeOptions: number[]
+  onSuccess?: (events: Log[]) => void
+}) => {
+  const { createSplit, error } = useCreateSplit()
+  // NOTE: should we log this error?
+  if (error) {
+    console.error(error)
+  }
+
```

Comment:
Helps to debug but not sure this is the best way to propagate error messages

Reply:
should we have an `onError` option for create split too?

Reply:
Yeah good idea